cernopendata/modules/theme/assets/semantic-ui/scss/frontpage.scss

```diff
@@ -221,7 +221,7 @@ $break-md: 768px;
   filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ffffff', endColorstr='#ededed', GradientType=1 );
 
   z-index: 999999;
-  height: calc(100vh-66px);
+  //height: calc(100vh-66px);
```

Comment:
This one was not working. It might be better to change the syntax instead of commenting it out